weave-js/src/components/PagePanelComponents/Home/Browse3/compare/hooks.ts

```diff
@@ -87,17 +87,12 @@ export const useObjectVersions = (
   // TODO: Need to introduce a backend query (/objs/read?) to bulk get specific versions of objects
   const {useRootObjectVersions} = useWFHooks();
   const objectIds = getUniqueNames(objectVersionSpecifiers);
-  const rootObjectVersions = useRootObjectVersions(
+  const rootObjectVersions = useRootObjectVersions({
     entity,
     project,
-    {
-      objectIds,
-    },
-    undefined, // limit
-    // TODO: This is super wasteful - getting all the data for all versions of every object mentioned
```

Comment:
lost comment?

Reply:
yeah sorry, i have to go through and add some bits back, i let claud off the chain for some of the refactors. I'm mostly looking for like high level comments while in draft :)